core/src/main/java/org/frankframework/compression/ZipIteratorPipe.java

```diff
@@ -177,8 +178,14 @@ public String next() throws SenderException {
 				log.debug("found zipEntry name [{}] size [{}] compressed size [{}]", current::getName, current::getSize, current::getCompressedSize);
 				String filename = current.getName();
 				if (isStreamingContents()) {
+					MessageContext context = new MessageContext().withName(current.getName()).withSize(current.getSize());
```

Comment:
Mogelijk dat het gebruik van `current.getSize()` hier een performance issue is?
Misschien beter om de `current.getCompressedSize()` te pakken?
Ik weet niet of het veel uitmaakt wel van de 2 wordt gebruikt voor message size.

Reply:
Ik denk dat we het dan beter weg kunnen laten, want anders klopt de daadwerkelijke message size niet.